addOns/jwt/src/main/resources/org/zaproxy/zap/extension/jwt/resources/Messages.properties

```
@@ -0,0 +1,81 @@
+jwt.settings.title=JWT Settings
+jwt.toolmenu.settings=JWT Settings...
```

Neither of these should need the word "Settings" they're only used on options panels.
```suggestion
jwt.settings.title=JWT
jwt.toolmenu.settings=JWT
```

Also the second one seems to suggest that it's a string used or displayed in the tools menu, which does not seem to be the case. In fact I think it's unused.

---

you are right. Will do the change